### ISSUE TITLE
USWDS - Future Bug: Banner references deprecated lock icon on Chromium browsers

### ISSUE THREAD

**Issue by werdnanoslen** [uswds/uswds#5270]
### Describe the bug

Around September, Chromium v117 [will replace the HTTPS padlock icon](https://blog.chromium.org/2023/05/an-update-on-lock-icon.html) with this "Tune" icon:
<img width="100" src="https://user-images.githubusercontent.com/764090/236246337-b3ef7449-e6a1-4604-95f9-6e242652193f.png"/>

Currently, the [banner](https://designsystem.digital.gov/components/banner/) references this lock icon.
<img width="442" alt="image" src="https://user-images.githubusercontent.com/764090/236247360-f1fed621-ad0e-460e-879d-c90ac106baba.png">

Chromium browsers include Chrome, Microsoft Edge, and Opera, so this affects a large majority of internet traffic. [Ars Technica expects](https://arstechnica.com/information-technology/2023/05/google-will-retire-chromes-https-padlock-icon-because-no-one-knows-what-it-means/) Safari and Firefox to follow suit.

### Steps to reproduce the bug

1. Wait til browsers make this change
2. On banners, click 'here's how you know'
3. See lock icon reference that doesn't mach what most people see

### Expected Behavior

Change references to something more generic (e.g. just the https) or directly reference the new way of accessing this information (tune icon).

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by werdnanoslen** [comment]
yep, this is just me getting the ticket in early to be prepared for the change

**Comment by dadrian** [comment]
I recommend switching the description to something along the lines of "you are insecure if the browser says 'Not Secure', otherwise everything is good".

(I wrote the Chromium blog post)
